packages/terra-application-utility/src/utility/_UtilityMenu.module.scss

```diff
@@ -5,11 +5,11 @@
   }
 
   .header-utility-menu {
-    background-color: var(--terra-application-header-utility-menu-background-color);
+    background-color: var(--terra-application-utility-header-menu-background-color);
   }
 
   .menu-utility-menu {
-    background-color: var(--terra-application-menu-utility-menu-background-color);
+    background-color: var(--terra-application-utility-menu-menu-background-color);
```

Comment:
Should `menu-menu` repeat?

Reply:
So Utility has two versions of itself, Menu and Header, and these versions have smaller parts of it. The first word after terra-application-utility is supposed to denote which type we're in, and everything after that denotes what part we are describing.  So menu type's menu background color, versus header type's menu background color, etc.

Reply:
It was pointed out by @emilyrohrbough that this is going to be a major version bump so we could probably use this as a chance to rename some these variables to not be repeats of two words.

Reply:
Let's leave this as is.

Reply:
Would changing it to one of these be better?
`--terra-application-utility-menu-utility-menu-background-color`
`--terra-application-utility-menu-type-menu-background-color`
I am leaning toward `type` just to make it slightly clearer.
@bjankord thoughts?

Reply:
Yeah, let's go with `type`, I think that adds some clarity to it.